external/implementations/samlconf-openam-impl/src/main/resources/OpenAMLoginData.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<data>
+
+    <!-- The OpenAM realm for the IDP -->
+    <realmName>/</realmName>
+
+    <!-- The openAM url -->
+    <baseUrl>http://localhost:8080/OpenAM-14.1.13</baseUrl>
+
+    <!-- Below is the default login form that gets sent to OpenAM -->
+    <!-- Add and remove input/output fields, using the below format, as needed to login to the IDP. -->
+    <loginForm>
+        <input-1>User Name:</input-1>
+        <output-1>ctk</output-1>
+        <input-2>Password:</input-2>
+        <output-2>gotestit</output-2>
```

Comment:
📘 Because users running the CTK can pass in -u username:password, I'm going to change these fields be defaulted to something that gets picked up by the parser.
IE: 
```
<input-1>User Name:</input-1>
<output-1>USER.ARG</output-1>
```